client/post-editor/media-modal/detail/detail-file-info.jsx

```diff
@@ -84,6 +84,21 @@ class EditorMediaModalDetailFileInfo extends React.Component {
 		);
 	};
 
+	renderFileSize = () => {
+		const fileSize = this.getItemValue( 'size' );
+
+		if ( ! fileSize || fileSize === 0 ) {
+			return;
+		}
+
+		return (
+			<tr>
+				<th>{ this.props.translate( 'File Size' ) }</th>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **141** times:
`translate( 'File size:' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/49500)